pkg/querier/worker/frontend_processor.go

```diff
@@ -14,6 +14,7 @@ import (
 	"github.com/go-kit/log"
 	"github.com/go-kit/log/level"
 	"github.com/grafana/dskit/backoff"
+	grpcUtils "github.com/weaveworks/common/grpc"
```

Comment:
Package names, and aliases, should be all lowercase, so `grpcutil`.

Also an opinionated nit, but in this case I'd call it `commongrpc`.